internal/api/mfa.go

```diff
@@ -244,7 +244,7 @@ func (a *API) VerifyFactor(w http.ResponseWriter, r *http.Request) error {
 		if err != nil {
 			return err
 		}
-		return badRequestError("%v has expired, verify against another challenge or create a new challenge.", challenge.ID)
+		return unprocessableEntityError(ErrorCodeMFAChallengeExpired, "MFA challenge %v has expired, verify against another challenge or create a new challenge.", challenge.ID)
```

Comment:
Potential legitimate HTTP status code change, though unlikely to be handled by developers.